wpeverest/everest-forms
# Enhancement - Global Setting Premium Sidebar.

### All Submissions:


### Changes proposed in this Pull Request:
Previously, there was no premium side bar design in Everest Forms this features. This PR enhances this feature. 

### How to test the changes in this Pull Request:

1. Go to global setting  of Everest Forms.
2.  There we can see sidebar in the right side of settings.

### Types of changes:




### Other information:




### Changelog entry

> Enhancement - Global Setting Premium Sidebar.
